premerge/post-submit-testing.md

```diff
@@ -0,0 +1,114 @@
+# Post Submit Testing
+
+## Introduction
+
+While this infrastructure is focused on premerge testing, it is also important
+to make sure that the specific configuration we are testing is tested post
+commit as well. This document outlines the motivation for the need to test this
+configuration post commit, why we are utilizing this design over others, and
+how we plan on implementing this to ensure we get fast feedback scalably.
+
+## Background/Motivation
+
+It is important that we test the premerge configuration postcommit as well.
+This enables easily checking the state of `main` at any given time and also
+easily pinpointing where exactly `main` has broken which enables figuring
+out which commit to revert/fix forward to get everything back to green.
```

Comment:
Very long sentence. Maybe try flipping the order: not "This enables that enables the other", but We want "the other' (why?); in order to get the other we need "that", which means we need to do "this".